test/chart.test.ts

```diff
@@ -65,21 +70,21 @@ describe("xlsx-wasm test", () => {
       .setDataLabel(chartDataLabel1)
       .setMarker(chartMarker1);
 
-    const chartDataLabel2 = new ChartDataLabel();
+    const chartDataLabel2 = new ChartDataLabel().setFont(chartFont).showValue().setPosition(ChartDataLabelPosition.Left);
     const chartSeries2 = new ChartSeries();
-    const chartLine2 = new ChartLine();
-    const chartSolidFill2 = new ChartSolidFill();
-    const chartFormat2 = new ChartFormat();
-    const chartMarker2 = new ChartMarker();
+    const chartLine2 = new ChartLine().setColor(Color.purple());
+    const chartSolidFill2 = new ChartSolidFill().setColor(Color.purple());
+    const chartFormat2 = new ChartFormat().setLine(chartLine2).setSolidFill(chartSolidFill2);
+    const chartMarker2 = new ChartMarker().setType(ChartMarkerType.Diamond).setSize(10).setFormat(chartFormat2);
     const categoriesRange2 = ChartRange.newFromRange("Sheet1", 1, 0, 6, 0);
     const valuesRange2 = ChartRange.newFromRange("Sheet1", 1, 2, 6, 2);
     chartSeries2
       .setName("Score 2")
       .setCategories(categoriesRange2)
       .setValues(valuesRange2)
-      .setFormat(chartFormat2.setLine(chartLine2.setColor(Color.purple())).setSolidFill(chartSolidFill2.setColor(Color.purple())))
-      .setDataLabel(chartDataLabel2.setFont(chartFont).showValue().setPosition(ChartDataLabelPosition.Left))
-      .setMarker(chartMarker2.setType(ChartMarkerType.Diamond).setSize(10).setFormat(chartFormat2.setLine(chartLine2.setColor(Color.purple())).setSolidFill(chartSolidFill2.setColor(Color.purple()))));
+      .setFormat(chartFormat2)
+      .setDataLabel(chartDataLabel2)
+      .setMarker(chartMarker2);
```

Comment:
Please leave this test as-is.
The way this test is written is intentional, to verify that self-mutation is working.
ref https://github.com/estie-inc/wasm-xlsxwriter/issues/6

Reply:
Thanks, makes sense! I’ll keep this in mind when improving the tests.